tools/setupResourceScripts/suites/snapshot-request-model-by-request-id.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Is this used? 
The regular snapshot create method is embedded int the setup script. So, I think we might either want to either embed this request as well or pull them both into their own folder of JSON requests. 
Either way, probably getting out of scope of this PR.

Reply:
it is unused I'll delete!